2D3D_UnityProject/Assets/Scripts/Utility/UI/Dialogue Screen.prefab

```diff
@@ -0,0 +1,704 @@
+%YAML 1.1
```

Comment:
again this prefab should be in the Prefabs folder, prolly under something like Prefabs/UI/Dialogue/